frontend/app/routes/_public+/apply+/$id+/terms-and-conditions.tsx

```diff
@@ -0,0 +1,173 @@
+import { useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import { ButtonLink } from '~/components/buttons';
+import { CollapsibleDetails } from '~/components/collapsible';
+import { InlineLink } from '~/components/inline-link';
+import { getApplyFlow } from '~/routes-flow/apply-flow';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { RouteHandleData } from '~/utils/route-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('terms-and-conditions');
+
+export const handle = {
+  breadcrumbs: [
+    {
+      labelI18nKey: 'terms-and-conditions:terms-and-conditions.index.breadcrumbs.terms-and-conditions',
+    },
+  ],
+  i18nNamespaces,
+  pageTitleI18nKey: 'terms-and-conditions:terms-and-conditions.index.page-title',
+} as const satisfies RouteHandleData;
+
+export async function loader({ request, params }: LoaderFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id, state } = await applyFlow.loadState({ request, params });
+  return json({ id, state: state.termsAndConditions });
+}
+
+export async function action({ request, params }: ActionFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id } = await applyFlow.loadState({ request, params });
+
+  const sessionResponseInit = await applyFlow.saveState({
+    request,
+    params,
+    state: {},
+  });
+
+  return redirect(`/apply/${id}/personal-info`, sessionResponseInit);
+}
+
+export default function TermsAndConditions() {
+  const { id } = useLoaderData<typeof loader>();
+  const { t } = useTranslation(i18nNamespaces);
+  return (
+    <>
+      <p className="text-xl">{t('terms-and-conditions:terms-and-conditions.index.intro-text')}</p>
+      <br />
+      <CollapsibleDetails id="collapsable-description-first-part" summary={t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-title')}>
+        <p>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-one')}</p>
+        <br />
+        <p>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-two')}</p>
+        <br />
+        <p>
+          <strong>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-three-bold')}</strong>
+          {t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-three-text')}
+        </p>
+        <br />
+        <p>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-four')}</p>
+        <br />
+        <p>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-five')}</p>
+        <br />
+        <p>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-six')}</p>
+        <br />
+        <strong>
+          <p>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-terms-of-use-of-the-online-application-title')}</p>
+        </strong>
```

Comment:
I feel like all these `*-title` i18n keys should be headings instead.

```suggestion
          <h2 className="font-bold">{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-terms-of-use-of-the-online-application-title')}</h2>
```

Reply:
Fixed.

Reply:
I misread thinking you want me to rename title to headings. Fixed.